specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1906,6 +2226,347 @@
+        },
+        "radiusSectorData": {
+          "type": "array",
+          "description": "Contains the information needed to plot wind radius quadrants. Bearing 0–90 = NE quadrant; 90–180 = SE quadrant; 180–270 = SW quadrant; 270–360 = NW quadrant.",
+          "items": {
+            "$ref": "#/definitions/RadiusSector"
+          }
+        },
+        "radiiGeometry": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/GeoJsonGeometry",
+          "description": "GeoJSON object. Displayed when radiiGeometry=true in request. Describes the outline of the wind radius quadrants."
+        }
+      }
+    },
+    "RadiusSector": {
+      "type": "object",
+      "description": "Bearing 0–90 = NE quadrant; 90–180 = SE quadrant; 180–270 = SW quadrant; 270–360 = NW quadrant.",
+      "properties": {
+        "beginBearing": {
+          "type": "number",
+          "format": "double",
+          "description": "Bearing, in degrees, of the beginning of the quadrant."
+        },
+        "endBearing": {
+          "type": "number",
+          "format": "double",
+          "description": "Bearing, in degrees, of the end of the quadrant."
+        },
+        "range": {
+          "type": "number",
+          "format": "double",
+          "description": "The radius of the quadrant, in nautical miles."
+        }
+      }
+    },
+    "StormLocationsResponse": {
+      "type": "object",
+      "description": "Locations for an individual government-issued storm",
+      "properties": {
+        "results": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocationResponse"
+          },
+          "description": "Locations for an individual government-issued storm"
+        }
+      }
+    },
+    "LocationResponse": {
+      "type": "object",
+      "description": "Location for an individual Government-issued storm",
+      "properties": {
+        "dateTime": {
+          "type": "string",
+          "description": "Datetime the forecast is valid, displayed in ISO8601 format."
+        },
+        "location": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/LatLongPair",
+          "description": "Coordinates of the storm"
+        },
+        "maxWindGust": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Maximum wind gust speed associated with the storm. May be NULL."
+        },
+        "sustainedWind": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Maximum sustained wind speed associated with the storm. May be NULL."
+        },
+        "minimumPressure": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Minimum pressure associated with the storm. May be NULL."
+        },
+        "movement": {
+          "$ref": "#/definitions/WindDetails",
+          "description": "The storm movement information."
+        },
+        "status": {
```

Comment:
can status be enumed?

Reply:
The list of status is as below.
http://apidev.accuweather.com/developers/glossary
There are more than 20 possible enum.
Do you still think we should make it an enum?